src/benchmarks.zig

```diff
@@ -565,3 +649,50 @@ pub fn benchmarkCSV(
         }
     }
 }
+
+const Metric = struct {
+    name: []const u8,
+    unit: []const u8,
+    value: u64,
+    // used to deallocate the metric name which
+    // may be on the heap due to runtime formatting
+    maybe_allocator: ?std.mem.Allocator = null,
```

Comment:
More of a question: All the instances of `Metric` had an allocator set, so is there a need to have this field as optional?

Reply:
true - 85c037de